### PR TITLE
systemtests: fix plugin postgresql testrunner-*, old postgres `bc` call

### PR DESCRIPTION
**Pull request by bruno-at-bareos**
**Backport of PR https://github.com/bareos/bareos/pull/1947 to bareos-23**

Also to fix issue during build and testing the old `postgres` plugin, we replaced the calls of `bc` tool by simple sh expressions.

#### Please check

- [X] Short description and the purpose of this PR is present _above this paragraph_
- [X] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)
- [Boy Scout Rule](https://docs.bareos.org/DeveloperGuide/generaldevel.html#boy-scout-rule)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
- [x] Original PR https://github.com/bareos/bareos/pull/1947 is merged
- [x] Required backport PRs have been created
- [x] Correct milestone is set

##### Source code quality
- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR

##### Tests
- [x] Decision taken that a test is required (if not, then remove this paragraph)
- [x] The choice of the type of test (unit test or systemtest) is reasonable
- [x] Testname matches exactly what is being tested
- [x] On a fail, output of the test leads quickly to the origin of the fault

### CODE REVIEW

File: systemtests/tests/py3plug-fd-postgresql/database/setup_local_db.sh.in
Comment (joergsteffens) [inline systemtests/tests/py3plug-fd-postgresql/database/setup_local_db.sh.in:1]: There is no need (anymore) that this file is a `*.in` file.

**Review by joergsteffens — CHANGES_REQUESTED** [review]
Generally this looks good, only two minor remarks left.

File: systemtests/tests/py3plug-fd-postgresql/testrunner-debian [hunk 1]
```diff
@@ -26,19 +27,25 @@ if ([ -z "${ID}" ] || [ "${ID}" != "debian" ] && [ "${ID}" != "ubuntu" ]);then
...
+if [ $UID -ne 0 ]; then
+# skip test
+ echo "${TestName} test skipped: needs to be root to use pg_*cluster system tools"
+ exit 77;
+fi
 
-PGVER=$(psql -V | awk '{gsub(".[0-9]$","",$3);print $3}')
-if [ ${PGVER} -lt 10 ]; then
+# We need the server version that will be used
+PG_VERSION=$(pg_lsclusters | awk -F " " '/^[0-9]{1,} .*/ {print $1}')
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply systemtests/tests/py3plug-fd-postgresql/testrunner-debian:37]: `psql` version is not really useful as the tool can connect to any pg => to it.
Your deterministic approach with `pg_lsclusters` while still perfectible is more what we want to test here. I will upport this to #1947 first then include it here as backport.

File: systemtests/tests/py3plug-fd-postgresql/database/setup_local_db.sh.in
Comment (bruno-at-bareos) [reply systemtests/tests/py3plug-fd-postgresql/database/setup_local_db.sh.in:1]: Nice catch, will do that also in #1947 and make the backport here.

Comment (bruno-at-bareos) [reply systemtests/tests/py3plug-fd-postgresql/database/setup_local_db.sh.in:1]: done

File: systemtests/tests/py3plug-fd-postgresql/testrunner-debian (hunk 1 above)
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply systemtests/tests/py3plug-fd-postgresql/testrunner-debian]: fixed

**Review by joergsteffens — APPROVED** [review]

**Pull request merged**